packages/components/namespace-store/namespace-store-form.tsx

```diff
@@ -267,10 +295,13 @@ const NamespaceStoreForm: React.FC<NamespaceStoreFormProps> = withHandlePromise<
           >
             <TextInput
               id="folder-name"
-              onChange={setFolderName}
-              value={folderName}
+              onChange={(e) =>
+                formDataDispatch({ type: 'setPVCFolderName', value: e })
```

Comment:
create enum for ActionType and use here while dispatching and in reducer file as well...